pkgs/tools/typesetting/tex/pplatex/default.nix

```
@@ -0,0 +1,49 @@
+{
+	pkgs,
+	fetchFromGitHub,
+	stdenv,
+	cmake,
+	pkgconfig,
+	pcre,
+	...
+} :
+	stdenv.mkDerivation {
+		pname = "fop";
+		version = "1.0-rc2";
+
+		src = fetchFromGitHub {
+			owner = "stefanhepp";
+			repo = "pplatex";
+			rev = "25bf7e121178f1fee2452b4fa9961248a045a387";
+			sha256 = "0xw7nvi2l15iyp9sm8vmmqghi54v99bcivqvx89f5v2gw0kw47k3";
+		};
+
+		nativeBuildInputs =
+		[
+			cmake
+			pkgconfig
+		];
+
+		buildInputs =
+		[
+			pcre
+		];
+
+		installPhase = ''
+			mkdir -p $out/bin
+			cp src/pplatex $out/bin
+		'';
+		
+
+		meta = {
+			description = "TeX/LaTeX family error parser";
```

Can use the description from the GitHub repo
```suggestion
			description = "A tool to reformat the output of latex and friends into readable messages";
```
Can drop `longDescription` then.